### ISSUE TITLE
training loss is NaN

### ISSUE THREAD

**Issue by ljjcoder** [ucbdrive/few-shot-object-detection#98]
Hi,
Thank you for the great work！I have configured the environment according to your installation document. In order to adapt to my native environment, I used the following command to install detectron2 adapted to cuda10.1 and torch1.6.

python3 -m pip install detectron2 -f https://dl.fbaipublicfiles.com/detectron2/wheels/cu101/torch1.6/index.html

Then I used the following command to train the model
CUDA_VISIBLE_DEVICES=0,2 python3 -m tools.train_net --num-gpus 2  --config-file configs/PascalVOC-detection/split1/faster_rcnn_R_101_FPN_base1.yaml
But the model diverged during the first iteration. Can you give me some suggestions to fix this error?

here is the log of training：
![1615978674(1)](https://user-images.githubusercontent.com/40750189/111457099-be9d9b00-8752-11eb-92c3-89817ee8888c.png)

**Comment by tailororrr** [comment]
I also encountered this problem

**Comment by hqhQAQ** [comment]
modify the detectron2 version to 0.3 or 0.2

**Comment by tailororrr** [comment]
when I modify the detectron2 version to 0.3, everything is ok

**Comment by ljjcoder** [comment]
@hqhQAQ thanks for your suggestions！
